README.md

```diff
@@ -1,7 +1,17 @@
-# 12barsongset
+Prerequisites
+Before installation, please make sure you have already installed the following tools:
 
-This is a React PWA created to show a reordable list of songs.
+git
+Nodejs
 
-Every song has its own route and it is possible to increase and decrease lyrics size. The text automatically diplays on columns based on the space available in the viewport.
+🛠️ Installation Steps
+1)Fork the project
+2)Clone the forked project
+3)Navigate to the cloned project directory
+4)Install the dependencies with yarn install
+-> just type yarn ( To install the dependencies )
+5)  To start the server type yarn start 
+->( this command will start your server )
 
-This app has been created to be used during live sets on a tablet fixed to the mic stand.
\ No newline at end of file
+along with this I will add some screenshots of your project, that is enough ? or do you want anything else
```

Comment:
I think you forgot to remove this and keep what there was before :) 

Everything else looks good, please update and I will accept the PR

Reply:
ok sir , i will update soon

Reply:
Please push another commit with just the readme changes to this specific branch so we can merge the PR so it will be counted

Reply:
how to do that thing sir , i am new to this so i don't know

Reply:
First move to the correct branch. I don't know if the name is codewithnadeem14502:patch-1 or just patch-1. 

`git checkout patch-1`
Then redo the changes to the readme file, save, commit and push

or run the following to copy the readme from the other branch where you did the changes
`git checkout nadeem -- README.md`

then commit and push. 

To know in which branch you are you can always run `git status` or `git branch` your branch will be highlighted

Reply:
<img width="769" alt="Screenshot 2022-10-09 at 17 26 43" src="https://user-images.githubusercontent.com/36789/194768285-9d4cbd88-aaaa-4b0e-80e6-1c50cb5bc264.png">
Here you can see where I saw your branch name

Reply:
sir, I try above but, can you please come on an online meet like google meet or zoom or another, where we can discuss this
![Uploading Screenshot (301).png…]()
, it will take only 5 mins only hope you consider my request

Reply:
> <img alt="Screenshot 2022-10-09 at 17 26 43" width="769" src="https://user-images.githubusercontent.com/36789/194768285-9d4cbd88-aaaa-4b0e-80e6-1c50cb5bc264.png">
> 
> Here you can see where I saw your branch name

![Uploading Screenshot (301).png…]()

Reply:
> First move to the correct branch. I don't know if the name is codewithnadeem14502:patch-1 or just patch-1.
> 
> `git checkout patch-1` Then redo the changes to the readme file, save, commit and push
> 
> or run the following to copy the readme from the other branch where you did the changes `git checkout nadeem -- README.md`
> 
> then commit and push.
> 
> To know in which branch you are you can always run `git status` or `git branch` your branch will be highlighted

https://www.linkedin.com/in/mohd-nadeem-8128311bb/

Reply:
Hi, in which time zone are you? I am in UK and I am happy to show you what I meant about branches in a google meet call if you wish. 


Anyway to make the long story short you need to have 2 different branches, one for each ticket. Everytime you work on a feature you have to work on its own specific branch to keep things in order. 
My previous suggestion was to reuse what you already have but if that is confusing for you try to follow the instructions below:

To create a branch you can just write `git checkout -b branch-name` where branch-name is the name of your branch e.g. update-readme or add-favicon

- `git checkout development` < you are positioning yourself on development so you get the original README
-  `git checkout -b feat/update-readme` < in this way you are creating a copy of development called feature/update-readme the magic happens thanks to the **-b** option

- Do your changes to readme only
- `git add .` < Add the modified files to the staged changes
- `git commit -m 'Add setup information to readme'` < commit and include a message, **-m** allows you to pass directly a string as message for the commit
- `git push` 

Then you can open the PR again.

I hope it helps a bit, I also added you on linkedIn so we can have a faster conversation.